web/lib/components/word/Speech.jsx

```diff
@@ -1,26 +1,50 @@
+import React from 'react'
 import { getVoiceSettings } from '../../storage'
 import { VolumnIcon } from '../common/Icons'
 
 const LANG = 'en-GB'
 
+function getVoices() {
+  return new Promise((resolve) => {
+    let interval
+
+    interval = setInterval(() => {
+      const voices = window.speechSynthesis.getVoices()
+      if (voices.length > 0) {
+        clearInterval(interval)
+        resolve(voices)
+      }
+    }, 10)
+  })
+}
```

Comment:
Let's rewrite this function and listen to `voiceschanged` event on `window.speechSynthesis` instead. It's more an assuring way.

https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesis/voiceschanged_event